### PR TITLE
Finishing project YoutubeClone HTML&CSS3 meeting requirements

### PR DESCRIPTION
**Pull request by lucianosarno**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    rel="stylesheet"
+  />  
+  <link
+  href="css/elements.css"
+  rel="stylesheet"
+  />
+  <link
+  href="css/highlight.css"
+  rel="stylesheet"
+  />
```
Comment (HADDADSOHAIB) [inline index.html]: There is no file highlight.css, so you need to remove this import.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      <img class="icon" alt="Account" src="images/avtar.jpg">  
+    </div>
+
+  </header>
+  <main >
+    <article class="underVideoContent">
+        <iframe id="iFrame" src="https://www.youtube.com/embed/XYeuvbhKy4I" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <h1 id="videoTitleH1">I Will Survive | Gloria Gaynor | funk cover ft. Mario Jose</h1>
+        <img id="viewsLikeShare" alt="Video information and sharing"  src="images/viewsLikesShare.jpg">
+        <img id="channelAndSubscribeImg" alt="Channel and subscribe button" src="images/channelAndSubscribe.jpg">
```
Comment (HADDADSOHAIB) [inline index.html]: It is not allowed to use images, you have to create the structure of the design using HTML and CSS

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @lucianosarno and @Prabhakarzx    :handshake: 

### Good job so far :clap:  your project looks like the original :medal_sports: 

### There are some changes to get the project approved :point_down:
- Please add a description to the pull request:
Current description:
Screenshot from 2020-06-09 01-13-12
Example of a good description in another project:
Screenshot from 2020-06-09 01-11-13
- [x] Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Please fix the issues with the validators:
Screenshot from 2020-06-09 01-14-59
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
- This part of the page is implemented using screenshots of the original page, that not allowed, you have to create the same design using HTML/CSS:
Screenshot from 2020-06-09 01-16-35
- [x] Ensure that students do not use screenshots of original page as shortcut to bypass requirements
- In your project, you divided the CSS of the same page into three files, that not a good practice, please put all the css of the same page into one file (index.css).
Screenshot from 2020-06-09 01-18-33
### Once you finish with implementing the requirements, submit for another code review request.
Good luck :+1:

**Comment by lucianosarno** [comment]
Hi Sohaib!
How are you?

Please, I would like to hear more about:

- The messages on validator aren't only warnings (instead of errors)? We
saw the messages but we thought that those places should not have H tags.
It is not allowed?

- Divide the css in 3 files isn't a good practice? Which consequences it
could have?
We did this way because it became pretty easier for conflict managing when
editing if we had different files for each order of priority, not?

About the other points, no doubts left.

Thanks in advance
Luciano

Em seg, 8 de jun de 2020 9:31 PM, Sohaib Haddad <notifications@github.com>
escreveu:

**Comment by HADDADSOHAIB** [comment]
Hi @lucianosarno  and @Prabhakarzx,

- For the validators, you must resolve all the error and warning, the validator make sure the quality of your code is good and meet the standard, it takes into consideration accessibility, SEO, semantic ..., for your case what I suggest is the following:
- Add h2...h6 to your section:
`<h3 class="hide">the main video</h3>`
- and in the CSS do the following:
`.hide {
  display: none;
}
`
- For the css files division, It is always bad to add additional files to the page, it slows down a load of your page, what you described to me can be achieved by putting all the CSS in one file and put the tags at beginning, classes in the middle and ids at the end (you can order them as you see fit).

-- Thanks for reaching out and if you have any additional feedback feel free to write back.

**Comment by lucianosarno** [comment]
Ok Sohaib thanks for clarifying!

Em ter, 9 de jun de 2020 6:57 AM, Sohaib Haddad <notifications@github.com>
escreveu:

**Comment by Prabhakarzx** [comment]
Thanks Sohaib!
Glad to see your detailed and constructive feedback.
We're working on it!
Thanks!

File: index.html (hunk 1 above)
Comment (Prabhakarzx) [reply index.html]: Thanks for the constructive feedback! Yes, we noticed that there was the file link left in the index.html. We used it for highlighting the page structure. Its removed now.
Thanks!

File: index.html (hunk 2 above)
Comment (Prabhakarzx) [reply index.html]: Thank You Haddad for your constructive feedback. We've worked on all the suggested improvements.
Thanks!

**Comment by Prabhakarzx** [comment]
Thanks Sohaib!
Thank you so much for the constructive feedback followed by the ways to counter the issues. We did it all. We've also fixed the warnings of header. 
Have a great day!
Thank You!

**Pull request closed without merging**